docs/code/readme_standards.md

```diff
@@ -3,100 +3,153 @@
 MDC Web component documentation serves two purposes:
 
 * Populate the [Catalog Site](https://material.io/components/web/catalog/)
-* Highlight how to use our APIs, for our external developers (see Usage section)
-
-> Developers should not have to read our code to know how to use our APIs
+* Highlight how to use our APIs, for our external developers.
 
 ## README Template
 
 Please follow the [README template](readme_template.md) *exactly*. This format is
 read/understood by the [Catalog Site](https://material.io/components/web/catalog/)
 
-## Usage
-
-Under the 'Usage' section, you will document all
+## Basic Usage
 
-* HTML Structure
-* CSS Classes
-* JavaScript
-* Sass Variables and Mixins
+The 'Basic Usage' section documents the bare minimum to get the most basic version of this component up and running, including:
+* HTML structure
+* Style import
+* JavaScript import and instantiation, if applicable
 
+~~~
 ### HTML Structure
 
-HTML Structure is documented with a code snippet, e.g.
-
 ```html
 <button class="mdc-button">Foo</button>
 ```
+~~~
 
-Any additional optional HTML should be documented in separate code snippets.
+In the 'Styles' subsection, include any imports and mixin invocations necessary to make the component look as desired.
+~~~
+### Styles
 
-### CSS Classes
+```scss
+@import "@material/button/mdc-button";
+
+// Include necessary mixin invocations here
+```
+~~~
+
+In the 'JavaScript Instantiation' subsection, include a note about importing a JavaScript component with a link to our importing documentation.
+~~~
+### JavaScript Instantiation
+
+```js
+import {MDCButton} from '@material/button';
+
+const button = new MDCButton(document.querySelector('.mdc-button'));
+```
+
+>See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
```

Comment:
Is this intentionally a blockquote? It isn't where we currently use it in docs.

If we want it to be a blockquote, nit: space after `>`

Reply:
I'm gonna make it a blockquote since it's less important now that we explicitly list the import statements here.